assets/components/signout/signout.jsx

```diff
@@ -25,22 +25,34 @@ function buildUrl(returnUrl: ?string): string {
 
 }
 
+function Signout(props: PropTypes) {
```

Comment:
Above this function, you could put the comment 
// ----- Component -----//

to be consistent with the rest of the files.